teachopencadd/tests/test_cli.py

```diff
@@ -0,0 +1,121 @@
+"""
+Unit tests for Command Line Interface.
+"""
+
+from pathlib import Path
+import subprocess
+
+from teachopencadd.utils import (
+    _greeting_string,
+    _run_jlab_string,
+    _talktorial_list_string,
+)
+from teachopencadd.cli import TALKTORIAL_FOLDER_NAME
+
+
+def capture(command):
+    """
+    Run input command as subprocess and caputure the subprocess' exit code, stdout and stderr.
+
+    Parameters
+    ----------
+    command : list of str
+        Command to be run as subprocess.
+
+    Returns
+    -------
+    out : bytes
+        Standard output message.
+    err : bytes
+        Standard error message.
+    exitcode : int
+        Exit code.
+    """
+
+    proc = subprocess.Popen(
```

Comment:
I think you can use `universal_newlines=True` to decode the stdout/err and avoid using `b""` strings.